webrtc.html

```diff
@@ -4429,7 +4430,9 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
 
           <dd>The <code><a>RTCIceTransport</a></code> has finished gathering,
           received an indication that there are no more remote candidates,
-          finished checking all candidate pairs and found a connection.</dd>
+          finished checking all candidate pairs and found a connection.
+          If consent checks [[!RFC7675]] subsequently fail, the state 
```

Comment:
If consent checks fail in "connected" and there are no successful candidate pairs left, the state transitions to "disconnected" if there are no candidate pairs left to check (but gathering is not complete or the last remote candidate has not been received), or to "checking" if there are candidate pairs left to check.